addons/api/tests/unit/serializers/host-set-test.js

```diff
@@ -12,12 +12,14 @@ module('Unit | Serializer | host set', function (hooks) {
       compositeType: 'static',
       description: 'Description',
       host_catalog_id: '123',
+      sync_interval_seconds: 1,
```

Comment:
Does `sync_interval_seconds` belongs to `static`, `plugin` or borth types? be aware this test is JUST for static.

Reply:
seems like only for type `plugin`! I'll add a new test case for this

Reply:
good catch! I updated serializer, mirage, and tests to include `sync_interval_seconds` only in type `plugin`